scripts/src/public/services/offer_contents.ts

```diff
@@ -123,7 +126,7 @@ export function isValid(offerContent: db.OfferContent, form: string | undefined)
 	return typeof answers === "object" && !Array.isArray(answers);
 }
 
-export function sumCorrectQuizAnswers(offerContent: db.OfferContent, form: string | undefined): number {
+export async function sumCorrectQuizAnswers(offerContent: db.OfferContent, form: string | undefined, acceptsLanguagesFunc?: ExpressRequest["acceptsLanguages"]): Promise<number> {
```

Comment:
Why doesn't this only pass the actual language that the poll was served in?
after you change the answers to ids, you wouldn't need to know this anyway

Reply:
It just feels broken that you might be served the poll in one language and somehow check the results in another one

Reply:
I don't want to add a language column to Order. This will all change, as you said, in a few days so it doesn't matter much.

Reply:
kk, but this makes that task even more important

Reply:
Anything that will make us do that task and not postpone it anymore is a good thing.